__mocks__/patient.mock.ts

```diff
@@ -20,7 +19,14 @@ export const mockPatientAlice: Patient = {
     dead: false,
     deathDate: null,
     causeOfDeath: null,
-    preferredName: null,
+    preferredName: {
+      uuid: '00000000-0000-0001-0000-000000000000',
+      display: 'Alice Johnson',
+      givenName: 'Alice',
+      familyName: 'Johnson',
+      familyName2: null,
+      voided: false,
+    },
```

Comment:
Not quite sure if there was a reason we weren't adding the preferred name directly to the mock?